bodhi/server/models.py

```diff
@@ -1506,6 +1506,21 @@ def get_testcase_karma(self, testcase):
                         bad += 1
         return bad * -1, good
 
+    def beautify_title(self, amp=False):
+        if len(self.builds) == 1:
+            return packagename_from_nvr(self, self.builds[0].nvr)
+        elif len(self.builds) == 2:
+            return " and ".join([packagename_from_nvr(self, build.nvr) for build in self.builds[:2]])
```

Comment:
I would prefer generating a list of packages' names on one line and joining and return on a second line. If you use change mentioned in my first comment you still save one line.